### PR TITLE
Fix signal handler to perform SSL connection to kill query

### PR DESCRIPTION
**Pull request by jrahman-zz**
Summary: The signal handler does not correctly use SSL when connecting
to the server in order to kill any outstanding query. Fix that.

Test Plan: Run mysql; execute SELECT SLEEP(500); Ctrl-C the process

### CODE REVIEW

File: client/mysql.cc [hunk 1]
```diff
@@ -2192,8 +2194,9 @@ static int read_and_execute(bool interactive)
     if ((named_cmds || glob_buffer.is_empty())
 	&& !ml_comment && !in_string && (com= find_command(line)))
     {
-      if ((*com->func)(&glob_buffer,line) > 0)
-	break;
+      if ((*com->func)(&glob_buffer,line) > 0) {
```
Comment (lth) [inline client/mysql.cc]: We usually try not to change more than what's needed.

**Review by lth — COMMENTED** [review]
File a bug upstream if you want.

File: client/mysql.cc [hunk 2]
```diff
@@ -4908,6 +4897,26 @@ sql_real_connect(char *host,char *database,char *user,char *password,
   return 0;
 }
 
+static void
+configure_ssl(MYSQL* mysql) {
+  #if defined(HAVE_OPENSSL) && !defined(EMBEDDED_LIBRARY)
```
Comment (lth) [inline client/mysql.cc]: Can we keep the same indentation as before? The #if should not be indented.

**Review by lth — APPROVED** [review]

**Comment by jrahman-zz** [comment]
Fixed indentation as requested. Imported and re-running tests now.

**Comment by jrahman-zz** [comment]
Fixed last indentation issue, should be ready

**Pull request closed without merging**